hwdb/70-mouse.hwdb

```diff
@@ -292,18 +292,18 @@ mouse:usb:v047dp1020:*Kensington Expert Mouse*
 # Lenovo
 ##########################################
 
-# Lenovo Optical USB Mouse
+# Lenovo USB mouse model MO28UOL
+mouse:usb:v04b3p310c:name:USB Optical Mouse:
+ MOUSE_DPI=400@142
+ 
+ # Lenovo Optical USB Mouse
```

Comment:
no preceding space here please,  that's probably why the CI fails

Reply:
Thanks! Fixed.